src/WaveClip.cpp

```diff
@@ -1567,20 +1493,16 @@ void WaveClip::ClearAndAddCutLine(double t0, double t1)
    }
 
    // Clear actual audio data
-   sampleCount s0, s1;
-
-   TimeToSamplesClip(t0, &s0);
-   TimeToSamplesClip(t1, &s1);
+   auto s0 = TimeToSequenceSamples(t0);
+   auto s1 = TimeToSequenceSamples(t1);
 
    // use Weak-guarantee
    GetSequence()->Delete(s0, s1-s0);
 
    // Collapse envelope
    auto sampleTime = 1.0 / GetRate();
    GetEnvelope()->CollapseRegion( t0, t1, sampleTime );
-   if (t0 < GetStartTime())
-      Offset(-(GetStartTime() - t0));
```

Comment:
I was wondering about these unexplained deleted lines... but I think we can prove that they never happened because of the lines in WaveTrack::HandleClear():

```
         if (!clip->BeforePlayStartTime(t1) && !clip->AfterPlayEndTime(t0) &&
               (clip->BeforePlayStartTime(t0) || clip->AfterPlayEndTime(t1)))
         {
            addCutLines = false;
            break;
         }
```

So, okay